packages/material-ui/package.json

```diff
@@ -24,7 +24,7 @@
     "build:es2015": "cross-env NODE_ENV=production babel ./src --out-dir ./build --ignore *.test.js",
     "build:es2015modules": "cross-env NODE_ENV=production BABEL_ENV=modules babel ./src/index.js --out-file ./build/index.es.js",
     "build:es": "cross-env NODE_ENV=production BABEL_ENV=es babel ./src --out-dir ./build/es --ignore *.test.js",
-    "build:umd": "cross-env NODE_ENV=production BABEL_ENV=production-umd rollup -c scripts/rollup.config.js && rimraf dist/material-ui.esm.js",
+    "build:umd": "BABEL_ENV=production-umd rollup -c scripts/rollup.config.js && rimraf dist/material-ui.esm.js",
```

Comment:
@oliviertassinari Feels like a mistake to remove `cross-env` here?

Reply:
Yes, we need cross-env.

Reply:
Do you want to submit a pull-request?

Reply:
Wasn't planning on it. I wasn't sure if it was intended, so I just asked. I'm kind of the opinion that you accepted the PR, you fix it. That said, I don't want to be considered a dick and I'm getting a lot of value from this project, so... #11638